app/portainer/views/groups/edit/groupController.js

```diff
@@ -23,6 +23,12 @@ function ($q, $scope, $state, $transition$, GroupService, TagService, Notificati
     });
   };
 
+  $scope.onCreateTag = async function onCreateTag(tagName) {
+    const tag = await TagService.createTag(tagName);
+    $scope.availableTags = $scope.availableTags.concat(tag);
+    $scope.group.TagIds = $scope.group.TagIds.concat(tag.Id);
+  }
```

Comment:
Wrap with non async function. The main goal for this wrapper is to trigger a digest cycle when the async function returns. See #2945 for more details.

Also, protect the await with a try/catch closure and show a notification if error occurs